### PR TITLE
Project Movies - Matilda Svensson & Jessika Hage

### PR DESCRIPTION
**Pull request by jessika-hage**
Link to my deployed project:
https://movies-time.netlify.app/

### CODE REVIEW

File: code/src/components/Error.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
```
Comment (jesinkey) [inline code/src/components/Error.js:1]: Nice that you created an error page!

File: code/src/components/FirstLoading.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
```
Comment (jesinkey) [inline code/src/components/FirstLoading.js:1]: Great touch with the first loading page

File: code/src/components/Footer.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+export const Footer = () => {
+  return (
+    <footer className="footer">
+      <div className="logos">
+        <i tabIndex='0' aria-label="Facebook logo" className='fa fa-facebook'></i>
```
Comment (jesinkey) [inline code/src/components/Footer.js:7]: Good work on aria-labels.

**Review by jesinkey — APPROVED** [review]
Everything looks great! Nice touches on the webpage with loading screens and fade-effects. A+!
(Sorry we were one day late)

**Review by KarolCislowski — APPROVED** [review]
Great project! I love the styling and animations 😻

**Pull request closed without merging**